swabbie-core/src/main/kotlin/com/netflix/spinnaker/swabbie/AbstractResourceTypeHandler.kt

```diff
@@ -752,6 +753,12 @@ abstract class AbstractResourceTypeHandler<T : Resource>(
     }
   }
 
+  private fun sendDeleteEvents(
```

Comment:
nit: this probably doesn't need to be a function because it's already one line: 
`resources.forEach { applicationEventPublisher.publishEvent(DeleteResourceEvent(it, workConfiguration))`

Reply:
updated